utils/httpUtils.ts

```diff
@@ -72,3 +71,114 @@ export const notFound = (message: string) => {
     message: message,
   });
 };
+
+export const getFeatureFlagsFromRequest = context => {
+  return context.request.headers.get("x-should-read-from-nextgen");
```

Comment:
is this async? if so do you mind adding a return type?

Reply:
oh nvm I see it's from the fetch

Reply:
yeah, this particular function is not async